Source/Urho3D/Network/Connection.cpp

```diff
@@ -1010,10 +1023,10 @@ float Connection::GetRoundTripTime() const
     return 0.0f;
 }
 
-float Connection::GetLastHeardTime() const
+unsigned Connection::GetLastHeardTime() const
 {
-    //TODO
-    return 0.0f;
+    Timer t = lastHeardTimer_;

```

Comment:
Don't think a copy is needed here.

Reply:
Otherwise compiler threw errors when the const method was trying to call `lastHeardTimer_.GetMsec(false)` method.

Reply:
Possible solution to this might be creating const method on Timer class which would return read only timer value in milliseconds.